cephfs/admin/bytecount.go

```diff
@@ -1,3 +1,4 @@
+//go:build !luminous && !mimic
```

Comment:
interesting new format, any idea what Go version introduced this (and when the old format can be dropped)?

Reply:
All I can say is, that the 1.16 gofmt did not add this, but 1.17 does. So for sure we could drop it when we stop supporting 1.16. possibly earlier, if earlier versions of the compiler support the new format as well.

Reply:
https://go.googlesource.com/proposal/+/master/design/draft-gobuild.md#transition
N=17